contents/blog/missing-recordings.md

```diff
@@ -0,0 +1,100 @@
+---
+date: 2021-06-10
+title: The mystery of missing session recordings at PostHog
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+author: ["rick-marron"]
+categories: ["Engineering", "Inside PostHog"]
+---
+
+[Session Recording](product/session-recording) is one of PostHog's core features. It allows you to see how real users use your website, mouse clicks and all. It's really cool and addictive (give it a try and you'll see what I mean).
+
+Recently, we made a ton of improvements to the feature. To name a few, we:
+
+* Decreased average recording load time from ~2.5s to 700ms
+* Resolved a big bug that caused recordings to "wonk out" if a recording spanned multiple tabs or browser windows
+* Revamped the player experience to make it faster for users to get to the moment in the recording they care about
+* Improved the speed to query recordings, allowing users to search across all recordings at once (it used to be limited to a single day's worth of recordings)
+
+But there was one fix that stood out in this quality push. Customers would often tell us recordings were missing, and we didn't know why.  
+
+This is the story of how we decreased the number of missing recordings from 15% to <1%, and in doing so, moved the feature out of beta.
+
+>[Rick Marron](/handbook/company/team#rick-marron-software-engineer) is a software engineer at PostHog. In previous lives Rick was a Program Manager at Microsoft, and he spent over a year in Nairobi, Kenya where he developed the first publicly available real-time source for traffic data in the city using computer vision and street cameras. He's a native of Rochester, New York but lives in Los Angeles now because... sunshine. Follow him [on Twitter](https://twitter.com/rick_marron) and [Github](https://github.com/rcmarron). 
```

Comment:
@rcmarron I've added this author bio (new thing I'm doing on the blog). Let me know if you're happy with it or change anything you don't like!

Reply:
Looks great to me!